### DISCUSSION TITLE
https://en.wikipedia.org/wiki/GrandOrgue

### DISCUSSION THREAD

**Discussion by jason-barba** [GrandOrgue/grandorgue#916] [Developers]
This wiki page needs some looking into.

I could help out, but I am not sure I have all the latest information...

//Jens

**Comment by larspalo** [comment]
Mainly the edits would need to be a link to this GitHub repository (and removal of the Sourceforge reference). Changing the main current developers to @larspalo, @oleg68 and @rousseldenis.

**Reply by jason-barba** [reply]
I can fix that.

//Jens

**Reply by oleg68** [reply]
> I can fix that.
> 
> //Jens

Great!. If would be nice.

**Reply by jason-barba** [reply]
Fixed.

**Reply by oleg68** [reply]
The urls are still pointing to the sourceforge instead of github

**Reply by jason-barba** [reply]
Mea culpa. Something went wrong during preview so I had to re-edit and thus failed to update the URLs.

**Reply by larspalo** [reply]
Maybe the links to my webpage should direct to http://familjenpalo.se/vpo/ so that it's immediately obvious what (and where) the relevant content are.

**Reply by jason-barba** [reply]
Fixed

**Comment by jason-barba** [comment]
I don't really appreciate the latest release info in the Infobox. This is data that has to be updated each time a new release is published. Would be good if this data was appended with a date, so the reader could draw conclusion of validity. "Latest release as off december 2021" for example.

I tried to add (as of dec 2021) to the latest release version, but it just looked weird.
